### PR TITLE
add a pr template

### LINKED ISSUE #847: Add a pull request template
Having a PR template can really help new contributors while raising a PR. With a PR template, new contributors have to worry less about adding proper descriptions to PR and rather focus on actually contributing new resources.

Also, it can help to keep all PRs consistent in the project.

### PR DESCRIPTION
**Pull request by Dharmik48**
Closes #847 

Added a pull request template for the repository.

### CODE REVIEW

**Review by donmarvex — COMMENTED** [review]
This looks great @Dharmik48 .

File: .github/pull_request_template.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+
+<!-- eg. [Resource Name](link) -> Use this format(including the square and round brackets) to add name link of resource -->
+
+## Short description
+
+<!-- Include a short description of the resource in this section -->
+
+## Why do you think this is a good resource?
+
+<!-- Add a reason that why the resource can be helpful to the community -->
```
Comment (donmarvex) [inline .github/pull_request_template.md:13]: I think "that" can go off from here. _Add a reason why the resource can be helpful to the community_ reads better.

Comment (Dharmik48) [reply .github/pull_request_template.md:13]: Let me fix that, just a second

Comment (donmarvex) [inline .github/pull_request_template.md:13]: I think "that" can go off from here. _Add a reason why the resource can be helpful to the community_ reads better.

Comment (Dharmik48) [reply .github/pull_request_template.md]: Done👍🏻 Removed _that_

**Review by donmarvex — COMMENTED** [review]
@sophiabrandt @kennithnichol could we get your thoughts on this pull request template for adding resources?

**Review by donmarvex — APPROVED** [review]

**Comment by sophiabrandt** [comment]
Nice idea, but we probably need a _prominent_ link (& how-to) to the template in the README, otherwise people won't understand what to do with it.

**Comment by Dharmik48** [comment]
Well, actually, GitHub will automatically pre-populate the PR description box when someone raises a PR.

So for example, I raise a PR then in the textbox where we add the description will be automatically filled with the contents of this template. 

So the contributors don't need any link to the actual template.

**Comment by donmarvex** [comment]
> Nice idea, but we probably need a _prominent_ link (& how-to) to the template in the README, otherwise people won't understand what to do with it.

I think the comments in the markdown file already sort of guides them on what to do. When a pull request is made, this raw markdown file is displayed so they can actually see the comments. They only need to fill in the necessary information and submit the PR.

**Comment by sophiabrandt** [comment]
> > Nice idea, but we probably need a _prominent_ link (& how-to) to the template in the README, otherwise people won't understand what to do with it.
> 
> I think the comments in the markdown file already sort of guides them on what to do. When a pull request is made, this raw markdown file is displayed so they can actually see the comments. They only need to fill in the necessary information and submit the PR.

Ah, alright, I didn't know that this will be prefilled automatically.

**Comment by Dharmik48** [comment]
> Ah, alright, I didn't know that this will be prefilled automatically.

Yeah! See, I created a new PR, and it was already filled with the template!

image

**Pull request merged**